tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -86,7 +86,9 @@ def test_fsx_lustre_configuration_options(
         storage_capacity=storage_capacity,
     )
     cluster = clusters_factory(cluster_config)
-    _test_fsx_lustre(cluster, region, scheduler, os, mount_dir, bucket_name, storage_type, auto_import_policy)
+    _test_fsx_lustre(
+        cluster, region, scheduler, os, mount_dir, bucket_name, storage_type, auto_import_policy, deployment_type
+    )
```

Comment:
not related to your change but could you please update the comment in this file (it's in 2 places):
`# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2`
--> 
`# FSx is only supported on ARM instances for Ubuntu 18.04, Amazon Linux 2 and CentOS 8`

Reply:
Done